scripts/packages/VSCodeServer/src/debug_cmds.jl

```diff
@@ -1,3 +1,16 @@
+const repl_startdebugger_notification_type = JSONRPC.NotificationType("repl/startdebugger", String)
+
+function repl_startdebugger_request(conn, params::String, crashreporting_pipename)
+    hideprompt() do
+        debug_pipename = params
+        try
+            DebugAdapter.startdebug(debug_pipename)
+        catch err
+            DebugAdapter.global_err_handler(err, catch_backtrace(), crashreporting_pipename, "Debugger")
+        end
+    end
+end
+
```

Comment:
Not these functions is better to live here ... Maybe renaming `debug_cmd.jl` into `debugger.jl` or something would make more sense ?

Reply:
Yes, agreed!